src/popup/pages/Popups/Connect.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <Modal
+    show
     full-screen
     class="connect"
     data-cy="popup-aex2"
   >
     <TransactionInfo
       :custom-title="$t('pages.connectConfirm.title')"
-      :sender="{ name: appName, address: app.host, url: app.url }"
+      :sender="{ name: appName, address: app?.host, url: app?.url }"
```

Comment:
I suggest to move the data passed to the prop to the computed property and set the return type to enforce the proper structure.